scripts/record_bag.py

```diff
@@ -110,12 +109,13 @@
     '/apollo/sensor/lidar16/rear/right/PointCloud2',
     '/apollo/sensor/lidar16/fusion/PointCloud2',
     '/apollo/sensor/lidar16/fusion/compensator/PointCloud2',
-    '/apollo/sensor/velodyne64/compensator/PointCloud2',
     '/apollo/sensor/lidar128/PointCloud2',
     '/apollo/sensor/lidar128/compensator/PointCloud2',
     '/apollo/sensor/lidar16/Scan',
     '/apollo/sensor/lidar16/PointCloud2',
     '/apollo/sensor/lidar16/compensator/PointCloud2',
```

Comment:
Do we need to put `/apollo/audio_detection` here too?

Reply:
yes, you are right.